sdk/requests/logout.js

```diff
@@ -0,0 +1,63 @@
+import { Linking } from 'react-native';
+import { getParameters, clearParameters } from '../configuration';
+import { logoutEndpoint } from '../utils/endpoints';
+
+const logout = async () => {
+  const parameters = getParameters();
+  const missingParams = 'Missing required parameter(s): ';
```

Comment:
```suggestion
  const missingParamsMessage = 'Missing required parameter(s): ';
```
Para no confundirnos con el objeto que tiene los parámetros. Si se acepta la sugerencia, es necesario cambiar la constante en todo el archivo.